GA-worksheet.md

```diff
@@ -7,17 +7,22 @@
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
+    To contain elements of headings.
```

Comment:
More specifically, the header tag actually holds the introductory material for the website, such as a logo and navigational links. Check it out on Buzzfeed.com. The top of the page that lets you know that you're on Buzzfeed is the header (inspect element to clarify).